### ISSUE TITLE
wrong `extra_info` when using `skip_segments`

### ISSUE THREAD

**Issue by TheGreatRefrigerator** [GIScience/openrouteservice#636]
#### Here's what I did

use this as request body for /directions/driving-car/geojson:
```
{"coordinates":[[8.681495,49.41461],[8.626507,49.41943],[8.687872,49.720318],[8.681495,49.51461]],"extra_info":["steepness","suitability","surface","waycategory"],"skip_segments":[2]}
```
---
#### Here's what I got

```
        "extras": {
          "waycategory": {
            "values": [
              [
                0,
                71,
                0
              ],
              [...],
              [
                85,  << No splitting at skipped segment
                502,  << it just gets swallowed
                0
              ],[...],[...],[...],
              [
                706,
                885, <<< last node extra_info gets computed for actual last is 886
                0
              ]
            ],
            "summary": [
              {
                "value": 0,
                "distance": 37443.9,
                "amount": 49.44   <<<< amount not adding up to 100 %
              },
              {
                "value": 1,
                "distance": 34224.7,
                "amount": 45.19 <<<
              },
              {
                "value": 2,
                "distance": 4065.5,
                "amount": 5.37 <<<<
              }
            ]
          },
        "way_points": [
          0,
          166,  << This is the start point of the skipped segment
          167, << endpoint of the skipped segment
          886 << actual last node number
        ]
```
---
#### Here's what I was expecting

something like 

        "extras": {
          "waycategory": {
            "values": [
              [
                0,
                71,
                0
              ],
              [...],
              [
                85,
                166,
                0
              ],  <<<<<<  integration of the skiped segment
              [
              166,
              167,
              -1  <<<<<< a defining value
              ],<<<<<<
              [
                167, 
                503,  
                0
              ],[...],[...],[...],
              [
                707,
                886, <<< calcualtion for every node
                0
              ]
            ],
            "summary": [
              {
                "value": -1,    <<<< summary entry with distance and amount of freehand segment
                "distance": x,
                "amount": y = (100 - 49.44 - 25.19 - 5.37) <<<  amount adding up to 100 % of the route
              },
              {
                "value": 0,
                "distance": 37443.9,
                "amount": 49.44   
              },
              {
                "value": 1,
                "distance": 14224.7,
                "amount": 25.19 
              },
              {
                "value": 2,
                "distance": 4065.5,
                "amount": 5.37
              }
            ]
          },
        "way_points": [
          0,
          166,
          167, 
          886
        ]
```
---
#### Here's what I think could be improved

The improvment suggestions inline in the above code example

**Comment by TheGreatRefrigerator** [comment]
I just realized a value -1 is not very good as it exists in steepness.
I'm not sure what would be applicable. Maybe 'false' or -999 or something

**Comment by aoles** [comment]
`NA`

**Comment by aoles** [comment]
Apparently [waycategory](https://github.com/GIScience/openrouteservice-docs#waycategory) is encoded on bit fields, so probably it shouldn't be a positive integer. `-1` sounds good to me though, I don't entirely follow why the value clash with steepness would be of an issue here. These are completely independent extra info entities, right?

**Comment by aoles** [comment]
Apparently various [extra infos](https://github.com/GIScience/openrouteservice-docs#routing-response) are encoded as integers, so we might want to stick to this. A large negative value sounds reasonable to me.

**Comment by TheGreatRefrigerator** [comment]
> 
> 
> Apparently [waycategory](https://github.com/GIScience/openrouteservice-docs#waycategory) is encoded on bit fields, so probably it shouldn't be a positive integer. `-1` sounds good to me though, I don't entirely follow why the value clash with steepness would be of an issue here. These are completely independent extra info entities, right?

I guess you would want the same "no-extra-info" value for each extra info available.
As -1 is an actual returned value for [steepness](https://github.com/GIScience/openrouteservice-docs#steepness) you would have to choose a different "no-steepness" value. I would welcome a generic approach where each _extra-info-category_ has the same value for missing infos.

**Comment by aoles** [comment]
I agree, I realized that afterwards and updated my comment just before your reply :)

**Comment by koebi** [comment]
This is a problem with [`constructFreeHandRoute`](https://github.com/GIScience/openrouteservice/blob/d8c0d31a4b731bd0c85e490ae7ccf97dcabfacc8/openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java#L538) in `ORSGraphHopper` not setting an `ExtraInfoProcessor`, as is done by [`calcPaths`](https://github.com/GIScience/graphhopper/blob/945d054c928b304789ba2c06dbdbd55d1cb0ee11/core/src/main/java/com/graphhopper/GraphHopper.java#L1142).

I'm a bit unsure on how to proceed here, since there doesn't exist a "default" `ExtraInfoProcessor` that sets default values for requested extra_infos.
